test/smoke/generated/storage-resource-manager/samples-dev/acquireALeaseOnAShare.ts

```diff
@@ -32,7 +33,7 @@ async function acquireALeaseOnAShare() {
     leaseId: undefined,
     proposedLeaseId: undefined
   };
-  const options = { parameters: parameters };
+  const options: FileSharesLeaseOptionalParams = { parameters: parameters };
```

Comment:
I would like this to be like this instead, `{parameters: parameters}` is unnecessary

```typescript
const options: FileSharesLeaseOptionalParams = { parameters: parameters };
```

Reply:
changed